src/main/pages/che-7/contributor-guide/proc_developing-che-theia-plug-ins-using-che.adoc

```diff
@@ -74,7 +74,7 @@ image::extensibility/running-plug-in-hosted-mode.gif[link="{imagesdir}/extensibi
 .. Using the command palette, stop your *Hosted Plug-in* instance (if any) by running the `Hosted Plugin: Stop Instance` command.
 .. Start the *Hosted Plug-in* instance again in the debugging mode by running the `Hosted Plugin: Debug Instance` command.
 .. Switch to the *Hosted Instance* tab and perform the required actions to reach the code with the breakpoint.
-.. Go back to the *Main Theia* window, and use the *Debug* panel as you need.
+.. Go back to the *Main Che-Theia* window, and use the *Debug* panel as you need.
```

Comment:
This looks like a GUI element -- we need to observe what's the actual name of the window.

Reply:
AFAIK, this window has not a name. Not visible either in https://www.eclipse.org/che/docs/che-7/developing-che-theia-plug-ins/. So I believe this change is safe.